telemetry-values.yaml

```diff
@@ -0,0 +1,284 @@
+certmanager:
```

Comment:
what is the purpose of this new file?

Reply:
sorry, it's my local test file, will remote that.

Reply:
removed this file.